### ISSUE TITLE
Panel: toggle button missing aria attributes

### ISSUE THREAD

**Issue by TaneliTuomola** [primefaces/primeng#17456]
### Describe the bug

[Toggle button (p-button) aria-attributes (label, controls, expanded) ](https://github.com/primefaces/primeng/blob/master/packages/primeng/src/panel/panel.ts#L60) are not passed to the rendered component (button). Replace p-button with the native button or make p-button support all aria-attributes?

### Reason for not contributing a PR

- [ ] Lack of time
- [x] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://primeng.org/panel

### Environment

https://primeng.org/panel

### Angular version

19

### PrimeNG version

v19

### Steps to reproduce the behavior

1. Open [primeNg.org/panel](https://primeng.org/panel) 2. Inspect elements

### Expected behavior

Aria-attributes work

**Comment by TaneliTuomola** [comment]
Same problem with Datepicker component:
https://github.com/primefaces/primeng/issues/18405

**Comment by TaneliTuomola** [comment]
> This issue has been automatically marked as stale due to a lack of activity. **If this issue is still relevant to you, please leave a comment** so we can keep it open. We apologize for not being able to prioritize it sooner. If you have any new information or questions, please share them in your comment!

The problem still exists (v21). https://github.com/primefaces/primeng/blob/master/packages/primeng/src/panel/panel.ts#L60
